src/service/data_products/common_functions.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
I had to split out the functions into a separate module from the common modules, as this function imports ArangoStorage, but that module imports stuff from the common data products module, causing a circular import